### PR TITLE
Add code to control the Compressor

### PR DESCRIPTION
**Pull request by ParkerMeyers**

### CODE REVIEW

**Review by BrownGenius — CHANGES_REQUESTED** [review]
Do we need to disable the compressor, or is that automatically done with the robot is disabled?

File: src/main/java/bhs/devilbotz/subsystems/Gripper.java [hunk 1]
```diff
@@ -52,5 +59,13 @@ public void stop() {
   @Override
   public void periodic() {
     // This method will be called once per scheduler run
+    System.out.println(
+        pcmCompressor.getPressure() + "\n " + pcmCompressor.getPressureSwitchValue());
```
Comment (BrownGenius) [inline src/main/java/bhs/devilbotz/subsystems/Gripper.java:63]: We should get rid of this and put it on the dashboard, instead.

Comment (ParkerMeyers) [reply src/main/java/bhs/devilbotz/subsystems/Gripper.java:63]: Get pressure does not work, I forgot to remove it before committing. Will remove now

File: src/main/java/bhs/devilbotz/subsystems/Gripper.java [hunk 2]
```diff
@@ -14,13 +15,18 @@
...
  *
  * @since 1/25/2023
- * @author joshuamanoj
+ * @author joshuamanoj & ParkerMeyers
  */
 public class Gripper extends SubsystemBase {
-  DoubleSolenoid gripperSolenoid;
+  private final DoubleSolenoid gripperSolenoid;
+  private static final Compressor pcmCompressor =
+      new Compressor(Constants.GripperConstants.COMPRESSOR_CAN_ID, PneumaticsModuleType.CTREPCM);
```
Comment (BrownGenius) [inline src/main/java/bhs/devilbotz/subsystems/Gripper.java:23]: The Pneumatics type should also be in the Constants file, but a minor issue.

**Review by BrownGenius — APPROVED** [review]

**Pull request merged**